imports/plugins/included/actionableAnalytics/client/components/InventoryReport.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { registerComponent } from "@reactioncommerce/reaction-components";
+import { SearchInput } from "../components/input/SearchInput";
+import { InventoryTable } from "../components/Table/InventoryTable";
+
+export const InventoryReport = ({ productInventoryData, selectedTab, handleInventoryInputChange }) => {
+  return (
```

Comment:
Adding this here for future reference...it's not something to be actioned now. When you are only returning in a function, you can omit `return` and write the function as:
```js
export const MyFunction = () => (
  <div>
    ...
  </div>
)